cli/src/commands/auth.rs

```diff
@@ -0,0 +1,46 @@
+use crate::commands::Command;
+use crate::config::CliConfig;
+use crate::errors::{ExitCode, Fallible};
+use crate::telemetry::Session;
+use crate::terminal::{confirm, input};
+use log::{info, warn};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub enum Auth {
+    /// Setup your auth stuff
+    /// Requires using an User key which can be found here:
+    /// https://engine.apollographql.com/user-settings
+    Setup(Setup),
+}
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
```

Comment:
I don't think we need this rename_all here since we don't have any subscommands. We many not need the `[structopt]` at all